src/pages/developing/svelte-tutorial/step-2.mdx

```diff
@@ -0,0 +1,719 @@
+
+```html path=src/routes/index.svelte
+<Row class="landing-page-background landing-page__banner">
+  <Column lg="{16}">
+    <Breadcrumb noTrailingSlash>
+      <BreadcrumbItem href="/">Getting started</BreadcrumbItem>
+    </Breadcrumb>
+    <h1>Design &amp; build with Carbon</h1>
+  </Column>
+</Row>
+```
+
+You may notice that the styles look off. Don't worry, we'll fix these later.
+
+### Second row
+
+In our second row we'll need `Tabs` and `Button` components. We'll update the
+`carbon-components-svelte` import to:
+
+```javascript path=src/routes/index.svelte
+import {
+  Grid,
+  Row,
+  Column,
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  TabContent,
+  Tab,
+} from 'carbon-components-svelte';
+```
+
+Modify the second row to use the `Tab` component.
+
+```html path=src/routes/index.svelte
+
+<Row class="landing-page__r2">
+  <Column noGutter>
+    <Tabs>
+      <Tab label="About" />
+      <Tab label="Design" />
+      <Tab label="Develop" />
+      <div slot="content">
+        <Grid as fullWidth let:props>
+          <TabContent {...props}>
+            <Row class="landing-page__tab-content">
+              <Column md={4} lg={7}>7/16</Column>
+              <Column md={4} lg={{ span: 8, offset: 1 }}>8/16</Column>
+            </Row>
+          </TabContent>
+          <TabContent {...props}>
+            <Row class="landing-page__tab-content">
+              <Column lg={16}>
+                Rapidly build beautiful and accessible experiences. The Carbon
+                kit contains all resources you need to get started.
+              </Column>
+            </Row>
+          </TabContent>
+          <TabContent {...props}>
+            <Row class="landing-page__tab-content">
+              <Column lg={16}>
+                Carbon provides styles and components in Vanilla, React,
+                Angular, Vue and Svelte for anyone building on the web.
+              </Column>
+            </Row>
+          </TabContent>
+        </Grid>
+      </div>
+    </Tabs>
+  </Column>
+</Row>
+
+
+```
+
+<InlineNotification>
+
+**Note:** We're using the grid for the page layout, but we also need to apply
+the grid within the tab content. When doing so, make sure the nested grid has
+the expected `grid` > `row` > `col` DOM structure.
+
+</InlineNotification>
+
+Hold up! If you were to run
+[DAP](https://www.ibm.com/able/dynamic-assessment-plug-in.html) to check for
+accessibility violations, you'd see
+`Multiple navigation landmarks must have unique labels specified with aria-label or aria-labelledby`
+because both the `Breadcrumb` and `Tabs` components use `<nav>` elements. To
+fix, add `aria-label` to the `Breadcrumb` opening tag:
+
+<!-- prettier-ignore-start -->
+```html
+<Breadcrumb noTrailingSlash aria-label="Page navigation">
+```
+<!-- prettier-ignore-end -->
+
+Same goes for the `Tabs` opening tag:
+
+<!-- prettier-ignore-start -->
+```html
+<Tabs aria-label="Tab navigation">
```

Comment:
Tabs in completed step 2 tutorial differ from preview in the horizontal position, looks like it's missing these styles in the tutorial:
<img width="156" alt="image" src="https://user-images.githubusercontent.com/40550942/201198109-203df2c8-9868-4802-9d9f-6c5c97cd1abe.png">
[Preview](https://svelte-step-3--carbon-tutorial-svelte.netlify.app/):
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/201198174-87496e75-3ad8-4845-a9c4-ecdd0a53b094.png">
Tutorial:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/201198246-2b8c2f8d-910a-4ad3-acdd-76bd4040b3e8.png">